RecoPPS/RPixEfficiencyTools/README.md

```diff
@@ -1,3 +1,47 @@
+## Onboarding
+### Onboarding - how to setup workspace 
+1. login to ssh 
+```
+ssh <username>@lxplus.cern.ch

```

Comment:
add a '-XY' option, which enables graphics passthrough. It's useful for ROOT displays etc. and there's no big drawback in using it as default

Reply:
I mention it directly in the `browsing root files` section. It provides a clearer explanation of why -XY flags are needed here

Reply:
Yes, but there's no real reason to keep the two commands different and it might only lead the user to login once like this and then later log back in with the different settings, which is pointless. If these options are set at first, re-logging is not even needed